### DISCUSSION TITLE
how to low down the gpu cost when i only need  ui

### DISCUSSION THREAD

**Discussion by Jimmy2099** [o3de/o3de#13356] [Help Wanted] [answered]
here is the problem,I try to use o3de make 2d only software (not a game)by using ui editor,I don’t need the 3d parts, when I finish some part of my software ,I found the my software use a lot gpu,about 80%~90%,I try to disable the Atom rendering engine,but when i disable it,the ui canvas doesn’t work,how I suppose to low down my gpu cost. I only need the ui part and scrips part of the engine

![image](https://user-images.githubusercontent.com/25509024/203465438-5cd9f6c4-4497-4db9-b03c-eb5b0ac7c906.png)
![image](https://user-images.githubusercontent.com/25509024/203465833-89ab5aac-90fb-4106-8297-770c56a9a3e1.png)
![image](https://user-images.githubusercontent.com/25509024/203465335-921ff4fd-b727-4ecb-8541-ab5974cfb65f.png)
![image](https://user-images.githubusercontent.com/25509024/203465371-c33a5863-166a-4594-9303-dd9d14a345cd.png)
![image](https://user-images.githubusercontent.com/25509024/203465399-656acda2-c3c3-4931-beb1-c69e93485ff1.png)

**Comment by jeremyong-az** [comment]
Do you have vsync disabled by any chance?

**Comment by rgba16f** [comment, marked as answer]
If you are only using the UI features you will probably want to dig into cutting down the render pipeline to remove most of the 3d passes.
Pinging @antonmic & @VickyAtAZ

**Reply by Jimmy2099** [reply]
Thanks for your reply, I just found an example called o3de-atom-sampleviewer, this example also demonstrates how to cut down the rending pipeline, so I'm already trying to do this.
![image](https://user-images.githubusercontent.com/25509024/203630789-35014917-234a-41fd-b59c-b2eece642553.png)
